app/assets/stylesheets/layout_components/forms.scss

```diff
@@ -63,6 +63,93 @@ form {
   }
 }
 
+/* ------------- Switch (Toggle Button) Styles ------------- */
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 64px;
+  height: 30px;
+  float: right;
+
+  &-slider {
+    position: absolute;
+    cursor: pointer;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: $brand-red;
+    -webkit-transition: 0.3s;
```

Comment:
This project uses https://github.com/postcss/autoprefixer, so we shouldn't need to manually add vendored attributes. Autoprefixer should automatically add this to our CSS for us!

We could get rid of all `-webkit` and `-ms` attributes in this PR. What do you think?

Reply:
Awesome, good to know